http/src/request/application/get_guild_command_permissions.rs

```diff
@@ -0,0 +1,37 @@
+use crate::request::prelude::*;
+use twilight_model::{
+    application::command::permissions::GuildCommandPermissions,
+    id::{ApplicationId, GuildId},
+};
+
+/// Fetch command permissions for all commands from the current application in a guild.
```

Comment:
We don't seem to typically use the terminology "fetch" in the crate, it seems to be "get".